libp2p/protocols/pubsub/rpc/messages.nim

```diff
@@ -63,6 +63,8 @@ type
     graft*: seq[ControlGraft]
     prune*: seq[ControlPrune]
     idontwant*: seq[ControlIWant]
+    preamble*: seq[ControlIHave]
+    imreceiving*: seq[ControlIWant]
```

Comment:
Shouldn't these have their own type? `ControlPreamble` and `ControlImReceiving`?